### PR TITLE
refactored time display for local_date on calendar_live.html.leex

### PR DESCRIPTION
**Pull request by danieldrojas**

### CODE REVIEW

File: lib/bordo_web/live/calendar_live.html.leex [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <div class="bg-white overflow-hidden shadow rounded-lg">
     <div class="border-b border-gray-200 p-3 sm:p-4 sm:pl-5 flex items-center justify-between flex-wrap sm:flex-no-wrap ">
       <h3 class="text-xl leading-6 text-gray-700">
-        <strong><%= Timex.format!(@current_date, "%B", :strftime) %></strong> <%= Timex.format!(@current_date, "%Y", :strftime) %>
+        <strong><%= BordoWeb.Helpers.TimeHelper.local_date(Timex.now, :short) %></strong>
```
Comment (chevinbrown) [inline lib/bordo_web/live/calendar_live.html.leex:10]: This will move everything inside the `<strong>`tag, so the formatting will be different?

Comment (danieldrojas) [reply lib/bordo_web/live/calendar_live.html.leex:10]: Yes, I'll change it.

File: lib/bordo_web/views/admin/team_view.ex [hunk 2]
```diff
@@ -34,7 +34,7 @@ defmodule BordoWeb.Admin.TeamView do
         Active
         <%= feather_icon("check-circle", "text-green-800 ml-1") %>
       </div>
-      <div class="text-sm leading-5 text-gray-500">Last paid: <%= Timex.format!(last_paid_at, "%Y-%m-%d", :strftime) %></div>
+      <div class="text-sm leading-5 text-gray-500">Last paid: <%= BordoWeb.Helpers.TimeHelper.local_date(last_paid_at, :numeric) %></div>
```
Comment (chevinbrown) [inline lib/bordo_web/views/admin/team_view.ex:37]: You should be able to alias this.

File: lib/bordo_web/helpers/time_helper.ex [hunk 3]
```diff
@@ -14,4 +14,8 @@ defmodule BordoWeb.Helpers.TimeHelper do
   def local_date(date, :short) do
     Timex.format!(date, "%B %Y", :strftime)
   end
+
+  def local_date(date, :numeric) do
```
Comment (chevinbrown) [inline lib/bordo_web/helpers/time_helper.ex:18]: What is `numeric`? Maybe `full` would make sense?

Comment (danieldrojas) [reply lib/bordo_web/helpers/time_helper.ex:18]: numeric is just numbers as in 20-10-02

File: lib/bordo_web/live/calendar_live.html.leex (hunk 1 above)
Comment (danieldrojas) [reply lib/bordo_web/live/calendar_live.html.leex:10]: This is the only case I ran into that I need the year(2020...) and month (January-December) to be apart so the year won't be inside the <strong> tag. Any suggestions?

**Pull request closed without merging**